mps_fast_patch.py

```
@@ -0,0 +1,233 @@
+"""
+Fast MPS optimization for Chatterbox TTS that pre-computes rotary embeddings.
+"""
+
+import torch
+import torch.nn as nn
+import logging
+import math
```

Ruff (0.11.9)

`math` imported but unused

Remove unused import: `math`

(F401)